canonical-web-and-design/ubuntu.com
# [CUBE] Update thank you and contact us

## Done

- Update Contact us text based on [this](https://github.com/canonical-web-and-design/ubuntu.com/issues/9316#issue-816721537)
- Update Thank you text based on [this](https://github.com/canonical-web-and-design/ubuntu.com/issues/9317#issue-816724097)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube
- Left Click the "Apply for access" button at the bottom of the page and check the text is as [requested](https://github.com/canonical-web-and-design/ubuntu.com/issues/9316), do the same when you right click and open a new tab for the form. 
- Fill out a test form and check the Thank you message is as [requested](https://github.com/canonical-web-and-design/ubuntu.com/issues/9317#issue-816724097)


## Issue / Card

Fixes #9316 and #9317
